packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -96,6 +96,6 @@ export function errorObjectFromSocketError(socketError: IR11sSocketError, handle
     return createR11sNetworkError(
         message,
         socketError.code,
-        socketError.retryAfterMs,
+        socketError.retryAfterMs === undefined ? undefined : socketError.retryAfterMs * 1000,
```

Comment:
should this be called socketError.retryAfterSeconds?

Reply:
That's part of protocol, so change like that is possible, but needs to be staged.
That's what I alluded in my comment above - ODSP uses time in seconds in protocol, it's not required but would be great for R11s to be the same here